runtimes/pythonrt/pythonrt.go

```diff
@@ -0,0 +1,453 @@
+package pythonrt
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/fs"
+	"os"
+	"path"
+	"time"
+
+	"go.autokitteh.dev/autokitteh/internal/backend/logger"
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/internal/xdg"
+	"go.autokitteh.dev/autokitteh/sdk/sdkruntimes"
+	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+	"go.uber.org/zap"
+)
+
+var (
+	Runtime = &sdkruntimes.Runtime{
+		Desc: kittehs.Must1(sdktypes.StrictRuntimeFromProto(&sdktypes.RuntimePB{
+			Name:           "python",
+			FileExtensions: []string{"py"},
+		})),
+		New: New,
+	}
+	venvPath = path.Join(xdg.DataHomeDir(), "venv")
+	venvPy   = path.Join(venvPath, "bin", "python")
+)
+
+type pySVC struct {
```

Comment:
pySvc?

Reply:
Don't have a strong opinion on this one. Changed.